packages/backend/src/request/request-context-provider.spec.ts

```diff
@@ -31,15 +31,18 @@ describe('RequestContextProvider', () => {
 
     let app = new App();
     let requestContextProvider = new RequestContextProvider(app);
-    requestContextProvider
+    return requestContextProvider
       .withRequestContext(request, response, _sctx => {
-        expect(() => {
-          requestContextProvider.withRequestContext(request, response, () => Promise.resolve());
-        }).toThrow();
-        return Promise.resolve();
+        return Promise.resolve(
+          expect(() => {
+            return requestContextProvider.withRequestContext(request, response, () =>
```

Comment:
this one also not needed as the throwing is synchronous and expect toThrow can only detect sync throws